kuma/landing/jinja2/landing/homepage.html

```diff
@@ -58,7 +58,7 @@ <h1 class="highlight highlight-spanned"><span class="highlight-span">{{ _('Resou
 <div class="home-callouts">
   <h2 class="offscreen">{{ _('Featured') }}</h2>
   <div class="column-container center">
-    {% if waffle.switch('foundation_callout') %}
+    {%- if settings.FOUNDATION_CALLOUT %}
```

Comment:
It might be nice to use:
```python
FOUNDATION_CALLOUT = config('FOUNDATION_CALLOUT', False, cast=bool)
```
instead, so it can be set from the environment if desired. For local development via Docker, one could then change the value via the `.env` file, and for production/stage pushes we could change the value without creating a new Docker image for Kuma (we'd have to add some minimal code to our K8s files as well, but we can do that later if we decide we want to do that).

Reply:
👍, except in ``kuma/settings/common.py``, not in the template.

Reply:
Sorry, thanks @jwhitlock, yes I attached this to the wrong spot. I had meant to attach this comment to `kuma/settings/common.py`.